eZ/Publish/Core/Pagination/Pagerfanta/SearchResultAdapter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Pagination\Pagerfanta;
```

Comment:
Looks like custom code consumes methods from this interface. In this case it should be API. Given the context - entire pagerfanta impl. - which shouldn't even be part of Kernel, I'll accept for now, especially due to time constraints. But if you have time, fix it please.